lib/active_model/serializer/error_serializer.rb

```diff
@@ -0,0 +1,2 @@
+class ErrorSerializer < ActiveModel::Serializer
+end
```

Comment:
I'm not sure at this time if I want to assign any attributes, or just use it as a 'pass through' for the adapter to make use of the 'object' (see `def object_errors` in the adapter).